.github/ISSUE_TEMPLATE/000-crash-report.yml

```diff
@@ -0,0 +1,85 @@
+name: Crash Report
+description: Report a bug that crashes the game or prevents startup.
+labels: [ "status: unverified", "bug: crash" ]
+body:
+  - type: input
+    id: version
+    attributes:
+      label: GregTech CEu Version
+      description: The version of Gregtech CEu you were using when this bug was encountered. If you do not know where to find this, look for the mod jar file in the mods folder, or in the mods viewer in-game.
+      placeholder: "Example: v2.3.4"
+    validations:
+      required: true
+  - type: input
+    id: modpack
+    attributes:
+      label: Modpack Used
+      description: The modpack and its version used when this bug was encountered. If you were not using a modpack, please leave this field blank.
+      placeholder: "Example: GregTech Community Modpack v1.2.1"
+    validations:
+      required: false
+  - type: textarea
+    id: addons
+    attributes:
+      label: Addons Installed
+      description: The addons and their respective versions used when this bug was encountered. If you were not using addons, or do not know which are present (i.e. in a modpack), please leave this field blank.
+      placeholder: "Example: Gregicality Multiblocks v1.1.3"
+    validations:
+      required: false
+  - type: input
+    id: environment
+    attributes:
+      label: Environment
+      description: "How you were playing on the world. Typical answers include: Singleplayer, Open to LAN, Forge Server, Sponge Server, or Mohist Server."
+      placeholder: "Example: Singleplayer"
+    validations:
+      required: true
+  - type: dropdown
+    id: crash
+    attributes:
+      label: Side Crashed
+      description: Where the game crashed.
```

Comment:
This either needs an explanation of how to find out what side crashed, or an `unsure` option. As I feel most players will just think "I crashed" rather than "I crashed the client"